app/helpers/recaptcha_helper.rb

```diff
@@ -0,0 +1,46 @@
+# Helper methods for turning CMS features on/off selectively
+module RecaptchaHelper
+  include MainSiteHelper
+
+  def minimum_score( action )
+    name = "recaptcha_v3_#{action}_score"
+    setting( name.to_sym ) || ENV[ name.capitalize ] || 0.5
+  end
+
+  def verify_checkbox_recaptcha
+    # Used in spec/requests/user_spec.rb, to test fallback to checkbox
+    return false if ENV[ 'FAIL_RECAPTCHA' ].present?
+
+    # TODO: fix the tests so they can get here (might need to refactor?)
+    # :nocov:
+    verify_recaptcha_v2
+    # :nocov:
+  end
+
+  def verify_invisible_recaptcha( action )
+    # Used in spec/requests/user_spec.rb, to test fallback to checkbox
+    return false if ENV[ 'FAIL_RECAPTCHA' ].present?
```

Comment:
Does this ENV var only exist for testing? If so, we may be able to replace it with mocking or stubbing one of these helper methods in the test.